Yoast/YoastSEO.js
# Add Italian Flesch reading

## Summary

This PR can be summarized in the following changelog entry: Added Flesch Reading for Italian.

## Test instructions

This PR can be tested by following these steps:

* Link YoastSEO.js
* Don't forget to run a `grunt build:js`
* Set site language to Italian.
* Write an Italian text. 
* See whether the Flesch reading score appears.

Fixes #1170
